### DISCUSSION TITLE
Ghost can't start after AWS RDS CA update

### DISCUSSION THREAD

**Discussion by mtdjr** [docker-library/ghost#423] [General]
I've updated Ghost's RDS database to use the new CA (old CA expires 3/5/2020) and Ghost is unable to make an SSL connection now.

We are using ghost:2.38. I've also tried with ghost:3 - same error.

```
[2020-02-18 19:34:05] ERROR unable to get local issuer certificate
 unable to get local issuer certificate
 "Unknown database error"
 Error ID:
    500
 Error Code: 
    HANDSHAKE_SSL_ERROR
 ----------------------------------------
 DatabaseError: unable to get local issuer certificate
    ...
```

**Comment by pascalandy** [comment]
Hi,
I don't think the issue you are describing is related the this docker container image.

To manage your expectations, you are in the **docker project** for Ghost. We don't support Ghost here, only the Ghost docker image itself.

For any other questions, please take a look at: https://forum.ghost.org/

Thanks!

**Comment by mtdjr** [comment]
@pascalandy I assumed it was an issue with the base image used, but it was a Ghost configuration issue. `database__connection__ssl` was set to `Amazon RDS`. Setting to `true` resolved the issue. Closing.

**Comment by Realvincentyuan** [comment]
Have exactly the same issue when the CA of AWS RDS is updated to `rds-ca-rsa2048-g1`.

Below config seems to not work for the SSL connection, any idea?

```json
"database": {
  "client": "mysql",
  "connection": {
    "host": "your_cloud_database",
    "port": 3306,
    "user": "your_database_user",
    "password": "your_database_password",
    "database": "your_database_name",
    "ssl": "Amazon RDS"
  }
}
```
